src/sendError.js

```diff
@@ -0,0 +1,11 @@
+const sendError = (response, statusCode, errorMessages) => {
```

Comment:
Let's make it for all dara
```suggestion
const sendResponse = (response, statusCode, data) => {
```